nominal/nominal.py

```diff
@@ -355,20 +357,20 @@ def __getattr__(self, k: str) -> None:
 
     def __init__(
         self,
-        rid: str = None,
-        path: str = None,
-        paths: list[str] = [],
-        datasets: list[Dataset] = [],
-        properties: dict = {},
-        title: str = None,
+        rid: str | None = None,
```

Comment:
I added all these optionals because pyright in vscode wasn't happy with `str = None` - and for a client lib I think it's best to be explicit on the types